cli/dcoscli/util.py

```diff
@@ -23,7 +23,7 @@ def wrapper(*args, **kwargs):
         try:
             result = func(*args, **kwargs)
         except docopt.DocoptExit as e:
-            emitter.publish("Command not recognized\n")
+            emitter.publish("Invalid subcommand usage\n")
```

Comment:
This would be this same message for all subcommands if they are misused (auth, job, etc.), this is only different for subcommands which have their own binary, where the behaviour depends on the executable I think.

Some examples : 
```
(dcos-cli/cli) ➜  cli git:(dcos-community-95) dcos auth
Invalid subcommand usage

Usage:
    dcos auth --help
    dcos auth --info
    dcos auth --version
    dcos auth list-providers [--json]
    dcos auth login
        [--provider=<provider_id>] [--username=<username>]
        [--password=<password> | --password-file=<password_file>
        | --password-env=<password_env> | --private-key=<key_path>]
    dcos auth logout
(dcos-cli/cli) ➜  cli git:(dcos-community-95) dcos job --invalid
Invalid subcommand usage

Usage:
        dcos job --config-schema
        dcos job --help
        dcos job --info
        dcos job --version
        dcos job add <job-file>
        dcos job remove <job-id> [--stop-current-job-runs]
        dcos job show <job-id>
        dcos job update <job-file>
        dcos job kill <job-id> [<run-id>][--all]
        dcos job run <job-id> [--json]
        dcos job list [--json]
        dcos job schedule add <job-id> <schedule-file>
        dcos job schedule show <job-id> [--json]
        dcos job schedule remove <job-id> <schedule-id>
        dcos job schedule update <job-id> <schedule-file>
        dcos job show runs <job-id> [--run-id <run-id>][--json][--q]
        dcos job history <job-id> [--json][--show-failures]
```